static/js/vue-cdr-access/src/mixins/analyticsUtils.js

```diff
@@ -1,18 +1,27 @@
 export default {
     methods: {
         pageEvent(recordData) {
-            let collection = recordData.briefObject.parentCollectionName || '';
+            this.$gtag.event('record', {
+                'event_category': recordData.briefObject.parentCollectionId,
+                'event_label': `${recordData.briefObject.title}|${recordData.briefObject.id}`
+            });
+        },
 
+        matomoPageEvent(recordData) {
```

Comment:
What would you think of having pageEvent call matomoPageEvent rather than specifying the implementation everywhere we want to trigger an event? Same with having pageView call matomoPageView internally. Then we could probably roll back the updates to most of the other components when there are pageView and matomoPageView calls side by side.